diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 7a1a6fff166193e13451be551e5f59083af4c889..585213ac7bc69ba7e42a449408b1c976379dcd44 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -412,10 +412,6 @@ else
         mediametrics
 endif
 
-ifneq (,$(RELEASE_RANGING_STACK))
-    PRODUCT_PACKAGES += \
-        com.android.ranging
-endif
 
 ifeq ($(RELEASE_PROCESS_MEMORY_GUARDIAN_DAEMON),true)
   PRODUCT_PACKAGES += \
diff --git a/target/product/base_vendor.mk b/target/product/base_vendor.mk
index a73d0936dd5ca7fa493344410547db53bc2610c5..c005b8ae67ed1bf6f6fd849c615375894bd5dfaa 100644
--- a/target/product/base_vendor.mk
+++ b/target/product/base_vendor.mk
@@ -75,8 +75,7 @@ PRODUCT_PACKAGES += \
     selinux_policy_vendor \
     selinux_policy_odm \
     shell_and_utilities_vendor \
-    odm-build.prop \
-    vendor_tracing_descriptors \
+    odm-build.prop
 
 # libhealthloop BPF filter. This is in base_vendor.mk because libhealthloop must
 # be a static library and because the Android build system ignores 'required'
