diff --git a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
index 2057bee07991a65ece70519af583a57ff626da20..cd72666c0f423e1c8d88d90e72f306d1442ade44 100644
--- a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
+++ b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
@@ -297,10 +297,10 @@ public class LineageDatabaseHelper extends SQLiteOpenHelper{
         }
 
         if (upgradeVersion < 20) {
-            // Clear Settings.Global.UIDS_ALLOWED_ON_RESTRICTED_NETWORKS
+            // Clear "uids_allowed_on_restricted_networks"
             if (mUserHandle == UserHandle.USER_SYSTEM) {
                 Settings.Global.putString(mContext.getContentResolver(),
-                        Settings.Global.UIDS_ALLOWED_ON_RESTRICTED_NETWORKS, "");
+                        "uids_allowed_on_restricted_networks", "");
             }
             upgradeVersion = 20;
         }
@@ -310,15 +310,15 @@ public class LineageDatabaseHelper extends SQLiteOpenHelper{
             SQLiteStatement stmt = null;
             try {
                 stmt = db.compileStatement("SELECT value FROM secure WHERE name=?");
-                stmt.bindString(1, Settings.Secure.TETHERING_ALLOW_VPN_UPSTREAMS);
+                stmt.bindString(1, "tethering_allow_vpn_upstreams");
                 long value = stmt.simpleQueryForLong();
 
                 if (value != 0) {
                     Settings.Secure.putInt(mContext.getContentResolver(),
-                            Settings.Secure.TETHERING_ALLOW_VPN_UPSTREAMS, (int) value);
+                            "tethering_allow_vpn_upstreams", (int) value);
                 }
             } catch (SQLiteDoneException ex) {
-                // LineageSettings.Secure.TETHERING_ALLOW_VPN_UPSTREAMS is not set
+                // Lineage"tethering_allow_vpn_upstreams" is not set
             } finally {
                 if (stmt != null) stmt.close();
             }
