diff --git a/private/compat/202504/202504.ignore.cil b/private/compat/202504/202504.ignore.cil
index a45d73e50f6f0ad3760960033d883a04410d60e1..6625a5e1687e9db7847155054eecdad3635e01f3 100644
--- a/private/compat/202504/202504.ignore.cil
+++ b/private/compat/202504/202504.ignore.cil
@@ -43,4 +43,12 @@
     mmd_setup
     contextual_mode_service
     system_server_wrapfd
+device_extras
+glyph_app
+nt_chg_recovery
+proc_charger
+proc_simulated_temp
+sltntc
+sysfs_powershare
+
   ))
